src/ucp/api/ucp.h

```diff
@@ -2449,6 +2449,38 @@ ucs_status_ptr_t ucp_stream_send_nb(ucp_ep_h ep, const void *buffer, size_t coun
                                     unsigned flags);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Non-blocking stream send operation.
+ *
+ * This routine sends data that is described by the local address @a buffer,
+ * size @a count object to the destination endpoint @a ep. The routine is
+ * non-blocking and therefore returns immediately, however the actual send
+ * operation may be delayed. The send operation is considered completed when
+ * it is safe to reuse the source @e buffer. If the send operation is
+ * completed immediately the routine returns UCS_OK.
+ *
+ * @note The user should not modify any part of the @a buffer after this
+ *       operation is called, until the operation completes.
+ *
+ * @param [in]  ep          Destination endpoint handle.
+ * @param [in]  buffer      Pointer to the message buffer (payload).
+ * @param [in]  count       Number of elements to send.
+ * @param [in]  flags       Reserved for future use.
+ * @param [in]  param       Operation parameters, see @ref ucp_request_param_t
```

Comment:
can we specify `dt` param for this cal?

Reply:
technically yes, but not sure how it could be used